src/sentry/static/sentry/app/views/projectInstall/createProject.tsx

```diff
@@ -92,7 +92,7 @@ class CreateProject extends React.Component<Props, State> {
         <div>
           <FormLabel>{t('Project name')}</FormLabel>
           <ProjectNameInput theme={theme}>
-            <ProjectPlatformIcon monoTone platform={platform} />
+            <StyledPlatformIcon platform={platform} />
```

Comment:
Should we pass `size=20px` here? The java icon looks wrong here:

![image](https://user-images.githubusercontent.com/79684/92669032-434fbe00-f2c5-11ea-8f15-5e22d16da47e.png)

Reply:
Hmm I can try updating it in the next batch to have a higher contrast? I have new logos to add for serverless and connect anyways

Reply:
We should do something, as we're showing these at 16px in at least one pretty prominent spot (the project selector in the global header). Maybe we should just enforce a min-size of 20px everywhere?

Reply:
That's definitely affect a lot of locations. Here's what it looks like with a new treatment:

**Before and after (2 color versions with stroke width applied)**
![Screen Shot 2020-09-10 at 3 24 08 PM](https://user-images.githubusercontent.com/4830259/92816000-b74b9e00-f379-11ea-8783-134428b6922d.png)

Reply:
@billyvg we're going to adjust the icon itself to read better at smaller sizes.